src/libraries/System.Private.CoreLib/src/System/SpanHelpers.T.cs

```diff
@@ -2614,11 +2615,41 @@ internal static int IndexOfAnyValueType<T>(ref T searchSpace, T value0, T value1
             return -1;
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveOptimization)]
+        private static bool VectorContainsMatch<TValue>(Vector128<TValue> inputVector) where TValue : struct, INumber<TValue>
+        {
+            // The inputVector has all the bits of a matching element set.
+            if (AdvSimd.Arm64.IsSupported)
+            {
+                // We select a larger byte from the adjacent pair of bytes. Thus, if any of the 16 bytes of inputVector are set,
+                // representing a matched element, they get reflected in the bottom half (8 bytes) of the register.
+                // We then detect a match when any of the bottom 8 bytes are non-zero.
+                return AdvSimd.Arm64.MaxPairwise(inputVector.AsByte(), inputVector.AsByte()).AsUInt64().ToScalar() != 0;
```

Comment:
Ah, so should be handled with https://github.com/dotnet/runtime/pull/75999 right?

Reply:
ooh right. Didn't notice the PR.